src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -908,7 +910,9 @@ javadocInlineTag:
             | LINKPLAIN_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference description?
             | LITERAL_LITERAL (WS | NEWLINE | LEADING_ASTERISK | text)*
             | VALUE_LITERAL (WS | NEWLINE | LEADING_ASTERISK)* reference?
-            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)* description?
+            | CUSTOM_NAME (WS | NEWLINE | LEADING_ASTERISK)*
+              ({!isNextJavadocTag()}? LEADING_ASTERISK | htmlComment
+              | CDATA | NEWLINE | text | javadocInlineTag | htmlElement)*
```

Comment:
There are cases of nested javadoc tags and HTML within javadoc tags. They can't be handled without allowing other tokens. For example look at this:

* @customTag <a> href="https://github.com/checkstyle/chestyle/"</a> text